index.html

```diff
@@ -0,0 +1,193 @@
+              alt=""
+            />
+            <p class="container-mint__name">mint</p>
+          </div>
+          <div class="animated-container">
+            <img
+              class="animated-container__logo"
+              src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9Ii01OTguNSA5NzEuNSA1MCA1MCI+PGNpcmNsZSBjeD0iLTU3My41IiBjeT0iOTk2LjUiIHI9IjI1IiBmaWxsPSIjMmNhMDFjIi8+PHBhdGggZD0iTS01OTEuNSw5OTYuNWMwLDUuNCw0LjQsOS44LDkuOCw5LjhoMS40di0zLjZoLTEuNGMtMy40LDAtNi4yLTIuOC02LjItNi4yczIuOC02LjIsNi4yLTYuMmgzLjQgYzAsMCwwLDE4LjYsMCwxOC44YzAsMiwxLjYsMy42LDMuNiwzLjZ2LTI2LjJjMCwwLTMuOCwwLTcsMEMtNTg3LjMsOTg2LjctNTkxLjUsOTkxLjEtNTkxLjUsOTk2LjV6IE0tNTY1LjEsOTg2LjdoLTEuNHYzLjZoMS40IGMzLjQsMCw2LjIsMi44LDYuMiw2LjJzLTIuOCw2LjItNi4yLDYuMmgtMy40YzAsMCwwLTE4LjYsMC0xOC44YzAtMi0xLjYtMy42LTMuNi0zLjZ2MjYuMmMwLDAsMy44LDAsNywwYzUuNCwwLDkuOC00LjQsOS44LTkuOCBDLTU1NS41LDk5MS4xLTU1OS43LDk4Ni43LTU2NS4xLDk4Ni43eiIgZmlsbD0iI2ZmZiIvPjwvc3ZnPg=="
+              alt=""
+            />
+            <p class="container-quiq__name">quiqbooks</p>
+          </div>
+          <div class="animated-container">
+            <img
+              class="animated-container__logo"
+              src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9Ii01OTguNSA5NzEuNSA1MCA1MCI+PGNpcmNsZSBjeD0iLTU3My41IiBjeT0iOTk2LjUiIHI9IjI1IiBmaWxsPSIjZDUyYjFlIi8+PHBhdGggZD0iTS01NzkuMSwxMDA1LjdjLTIuOC0yLTUuNi00LjItOC4yLTYuNmMtMSwyLjItMS44LDQuNC0yLjYsNi42YzQuMiwzLjYsOS44LDcuNiwxMy44LDEwIGM1LTEzLDEyLjgtMjEuNCwyMC40LTI3LjRjLTEtMi0yLjItMy44LTMuOC01LjRDLTU2Ny43LDk4OC45LTU3NC4zLDk5Ni43LTU3OS4xLDEwMDUuN3oiIGZpbGw9IiNmZmYiLz48L3N2Zz4="
+              alt=""
+            />
+            <p class="container-turb__name">turbotax</p>
+          </div>
+        </nav>
+      </section>
+      <section class="container-form">
+        <form action=".header">
+          <fieldset>
+            <div class="header-form">
+              <h1 class="title-form">Create an Intuit account</h1>
+              <h2 class="description-form">
+                One account for Iintuit, including Mint
+                <a href="#">Learn more</a>
+              </h2>
+            </div>
+            <label class="label-email lbl-position">
+              Email address
+              <input type="email" name="email" required />
+            </label>
+            <label class="label-phone lbl-position">
+              Phone (recommended)
+              <input type="tel" name="phone" />
+            </label>
+            <span>Standard call, messging or data rates may apply</span>
+            <div class="container-password">
+              <div class="label-password lbl-position" tabindex="0">
```

Comment:
Remember to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). You guys forgot to use the correct tag here. Kindly fix that.